it/src/main/java/org/corfudb/universe/node/server/vm/VmCorfuServerParams.java

```diff
@@ -22,24 +15,11 @@
 @Getter
 @EqualsAndHashCode(callSuper = true)
 @ToString(callSuper = true)
+@SuperBuilder
```

Comment:
Should we maintain an order of using those annotations?

Reply:
Done, Thank you.